src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -105,9 +123,43 @@ class Task extends React.Component<Props, State> {
         );
     };
 
+    fetchTaskCollaborators = async () => {
+        const { id, api, task_links } = this.props;
+        const { entries } = task_links;
+        let fileId;
+
+        if (entries[0].target) {
+            fileId = entries[0].target.id;
```

Comment:
Do we need to return early if there is no fileId?

Reply:
Fixed